nuxeo-common/src/main/java/org/nuxeo/common/xmap/XValueFactory.java

```diff
@@ -220,6 +223,42 @@ public String serialize(Context context, Object value) {
         }
     };
 
+    public static final XValueFactory DURATION = new XValueFactory() {
+        public final Pattern durationSimpleFormat = Pattern.compile("(?:(\\d+)m)?(?:(\\d+)s)?(?:(\\d+)ms)?");
```

Comment:
It's a constant, so `static` and all-caps name.

Reply:
Indeed, I didn't want to put in outside of `DURATION` instance, but no reason for.